inherit.yaml

```diff
@@ -0,0 +1,9 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Resources:
+    NestedStack:
+      Type: 'AWS::CloudFormation::Stack'
+      Properties: 
+          TemplateURL: 'https://notearshpc-quickstart.s3.amazonaws.com/0.2.1/cfn.yaml'
```

Comment:
Super cool! let's add this as an example to the README as well

Reply:
done.